torch/testing/_internal/opinfo/core.py

```diff
@@ -2123,6 +2135,11 @@ def generate_elementwise_unary_tensors(op, *, device, dtype, requires_grad, **kw
         a = make_arg(shape)
         yield SampleInput(a, kwargs=op.sample_kwargs(device, dtype, a)[0])
 
+    make_arg_channels_last = partial(make_arg, stride_permutation=[0, 2, 3, 1])
+    yield SampleInput(
+        make_arg((2, 3, 4, 5)), kwargs=op.sample_kwargs(device, dtype, a)[0]
```

Comment:
Probably meant to call `make_arg_channels_last()` here, but see above comment if channels last tensors should be generated more like noncontiguous tensors or arbitrarily strided tensors: 

https://github.com/pytorch/pytorch/blob/ac25c210e5452d360fcc8cf5ea96c85756e3e370/torch/testing/_internal/opinfo/core.py#L2230